### ISSUE TITLE
Mac Pro 3,1: NVMe APFS volume not bootable unless SATA APFS volume also connected

### ISSUE THREAD

**Issue by ahihi** [dakanji/RefindPlus#161]
### RefindPlus Version

v0.13.3.AD Release

### Device Type

Apple Mac

### Problem Description

Mojave APFS volume on SATA drive (A), Mojave APFS volume on NVMe drive (B). B does not show up as bootable unless A is also connected.

works with Mojave x64_apfs.efi installed in RefindPlus's drivers folder.

### Problem Point

Before starting a loader or tool

### Affected Items

Mac OS Loader

### Debug Log

linked below

### Additional Context

[macrumors thread](https://forums.macrumors.com/threads/refindplus-opencore-by-mybootmgr.2231693/post-32087309) with more info and logs

**Comment by dakanji** [comment]
Please test the attached X286 file to see if it resolves the APFS issue.
[X286-BOOTx64.zip](https://github.com/dakanji/RefindPlus/files/11255246/X286-BOOTx64.zip)


Run with the nvme driver but without the apfs driver.
Share the log generated.

Thanks

**Comment by dakanji** [comment]
@drhaynes ... Just crossed my mind that the X286 file may bypass the issues with your MVC flashed GPU from [Issue 159](https://github.com/dakanji/RefindPlus/issues/159). Please try it if possible and share the log. The drivers mentioned here do not apply in your case.

Thanks

**Comment by ahihi** [comment]
i installed this as BOOTx64.efi, removed x64_apfs.efi, and indeed it now works! :)

log: [23h21v2946.log](https://github.com/dakanji/RefindPlus/files/11298748/23h21v2946.log)

i noticed one small issue though, the RefindPlus boot screen has shrunk a little, as if it is not able to use the full 1920x1080 resolution: 
![IMG_20230421_221947](https://user-images.githubusercontent.com/738615/233721340-e257edf7-dcc3-4afe-881d-ab6669069fbe.jpg)

**Comment by dakanji** [comment]
Please try the attached as with the previous to hopefully close out.
Also drop the NVME driver (in addition to dropping the APFS one) as `SupplyNVME` is active here.
[X288-BOOTx64.zip](https://github.com/dakanji/RefindPlus/files/11299550/X288-BOOTx64.zip)

Will like to confirm:
1. Whether the graphics size issue is still present
2. Whether the NVME volumes shows up in RefindPlus
3. Whether you can boot the instance on the NVME drive from OpenCore without safe boot etc issues.

Please share the log as well.

Thanks

**Comment by ahihi** [comment]
> Also drop the NVME driver (in addition to dropping the APFS one) as `SupplyNVME` is active here. 

just to confirm, you mean it will be active even if i dont add `supply_nvme` to my config?

**Comment by dakanji** [comment]
Correct. It is forced on regardless of the config setting.

**Comment by ahihi** [comment]
alright, tested now. log: [23h22p2156.log](https://github.com/dakanji/RefindPlus/files/11301450/23h22p2156.log)

> 1. Whether the graphics size issue is still present

it appears to be fixed.

> 2. Whether the NVME volumes shows up in RefindPlus

yes.

> 3. Whether you can boot the instance on the NVME drive from OpenCore without safe boot etc issues.

yes, except i do still get occasional safe boots. sometimes the OpenCore boot picker starts up with a `+` cursor instead of the usual `*`, and the timeout fails to trigger. i have to hit return to boot my OS, and it seems this scenario always causes a safe boot.

i have been experimenting with having different sets of my usual usb devices connected while booting, and while im not 100% sure yet due to the random nature of this issue, so far it looks like it only happens when my D-Link usb hub is connected during boot time (even if no devices are plugged into the hub).

**Comment by dakanji** [comment]
> occasional safe boots

Is this is only with SupplyNVME or also happens with the standalone driver?

**Comment by ahihi** [comment]
also with the standalone driver.

**Comment by dakanji** [comment]
> may be a separate issue altogether.

Could be.

Will tag the APFS related original issue as a known issue and deal with it later.

In the meantime, you can download and use MyBootMgr v091 again as I have added the Nvidia bootscreen patch to that.
So you can switch the the REL build with that and not generate a log every time. Clear the existing logs.

This only has the bootscreen fix so you will need the drivers. A pre-release build of RefindPlus with all the fixes (there was a small tweak to SupplyNVME in X287) will be added to MyBootMgr v092. 

Everything will ultimately be added to the next release of RefindPlus but will appear in MyBootMgr beforehand.
